packages/cfd/.travis.yml

```diff
@@ -0,0 +1,11 @@
+sudo: false
```

Comment:
we don't use travis anymore. It's safe to remove this

Reply:
Done